scripts/build/redhat/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "etcher-electron",
+  "productName": "Etcher",
+  "genericName": "Etcher",
+  "homepage": "https://etcher.io/",
+  "icon": "assets/icon.png",
```

Comment:
AFAICT this is the icon that gets displayed for the program e.g. in the applications menu?
I dunno enough about rpm to know if there's an additional icon that you can choose to display during installation? \*shrug\*  Maybe @Dark-Passenger knows... ;-)